client/cluster.go

```diff
@@ -228,13 +228,17 @@ func (c *ClusterClient) Destroy(deleteRemoteWorkspace bool, kubernetesCleanup bo
 
 	// Best effort to clean up volumes and loadbalancer
 	if kubernetesCleanup && c.kubectl.IsUp() {
-		c.kubectl.DeleteAll("persistentvolumeclaims", "--timeout=5s")
-
-		// TODO Make smarter that to delete all pods, try to delete only once with pvc
-		c.kubectl.DeleteAll("pod", "--grace-period=60")
+		if err := c.kubectl.DeleteAllTimeout("persistentvolumeclaims", 5); err != nil {
+			return fmt.Errorf("error deleting persistent volume claims: %w", err)
```

Comment:
Do we really want to fail as soon as something goes wrong here? Would it not be good to try to delete as much as possible? You could either just log the errors or you could gather the errors in a `multierror` stack if you want to fail in the end.

Reply:
Failing early is sound in all cases when this is executed manually. It's only when run in CI you want "best-effort" cleanup.